install/update.php

```diff
@@ -4836,33 +4845,41 @@ function do_computerremotemgmt_migration($migration) {
 function do_computerarch_migration($migration) {
     global $DB;
 
-   if (TableExists('glpi_plugin_fusioninventory_computerarches')) {
-      //Rename field in coputeroperatingsystems table
-      $a_table = [
-         'name'     => 'glpi_plugin_fusioninventory_computeroperatingsystems',
-         'renamefields' => [
-            'plugin_fusioninventory_computerarches_id' => 'operatingsystemarchitectures_id'
-         ]
-      ];
-      migrateTablesFusionInventory($migration, $a_table);
+   if (TableExists('glpi_plugin_fusioninventory_computerarchs')) {
```

Comment:
Table name was `computerarches`; see original change https://github.com/fusioninventory/fusioninventory-for-glpi/commit/06a5fba4448c780d3b47c5b9a9f7427d34a5758c